contracts/BaseJumpRateModelV2.sol

```diff
@@ -102,38 +102,43 @@ abstract contract BaseJumpRateModelV2 is InterestRateModel {
      * @param borrows The amount of borrows in the market
      * @param reserves The amount of reserves in the market
      * @param reserveFactorMantissa The current reserve factor for the market
+     * @param badDebt The amount of badDebt in the market
      * @return The supply rate percentage per block as a mantissa (scaled by EXP_SCALE)
      */
     function getSupplyRate(
         uint256 cash,
         uint256 borrows,
         uint256 reserves,
-        uint256 reserveFactorMantissa
+        uint256 reserveFactorMantissa,
+        uint256 badDebt
     ) public view virtual override returns (uint256) {
         uint256 oneMinusReserveFactor = MANTISSA_ONE - reserveFactorMantissa;
-        uint256 borrowRate = _getBorrowRate(cash, borrows, reserves);
+        uint256 borrowRate = _getBorrowRate(cash, borrows, reserves, badDebt);
         uint256 rateToPool = (borrowRate * oneMinusReserveFactor) / EXP_SCALE;
-        return (utilizationRate(cash, borrows, reserves) * rateToPool) / EXP_SCALE;
+        uint256 rate = ((borrows * EXP_SCALE) / (cash + borrows + badDebt - reserves));
+        return (rate * rateToPool) / EXP_SCALE;
```

Comment:
I feel like this would be more intuitive:

```suggestion
        uint256 incomeToDistribute = borrows * rateToPool;
        uint256 supply = cash + borrows + badDebt - reserves;
        return (incomeToDistribute * EXP_SCALE) / supply / EXP_SCALE;
```

Reply:
Maybe with extra parenthesis to avoid any potential confusion?
```
return ((incomeToDistribute * EXP_SCALE) / supply) / EXP_SCALE;
```

Reply:
I have done the changes:
```suggestion
        uint256 incomeToDistribute = borrows * rateToPool;
        uint256 supply = cash + borrows + badDebt - reserves;
        return incomeToDistribute / supply;
```

Commit: 9e5839640529b400a12a594e6904e3071bf327a2

Reply:
@chechu , In this way we have removed `/EXP_SCALE` and `*EXP_SCALE`.